models/officerDelta.go

```diff
@@ -0,0 +1,62 @@
+package models
+
+type OfficerDelta struct {
+	Officer   []Officer `json:"officers"`
+	CreatedAt string    `json:"created_at"`
```

Comment:
Should be createdTime

Reply:
Will update to CreatedTime as that's what comes out of the DB package.